src/lib/io/sol-iio.c

```diff
@@ -92,6 +92,10 @@ struct resolve_absolute_path_data {
 #define SYSFS_DEVICES_PATH "/sys/bus/iio/devices"
 #define SYSFS_DEVICE_PATH "/sys/bus/iio/devices/%s"
 
+#define CONFIGFS_IIO_TRIGGERS_PATH "/sys/kernel/config/iio/triggers/hrtimer"
+#define CONFIGFS_IIO_TRIGGER_PATH "/sys/kernel/config/iio/triggers/hrtimer/%s"
```

Comment:
Minor: these two constants relate to 'hrtimer' - I'd put HRTIMER on their name, to be explicit.